qubesbuilder/plugins/chroot_rpm/mock/centos-stream.cfg

```diff
@@ -111,7 +111,7 @@ gpgkey=file://{keys_dir}/RPM-GPG-KEY-qubes-{qubes_version}-centos
 gpgcheck=1
 repo_gpgcheck = 1
 exclude=qubes-release
-enabled=0
+enabled={use_qubes_repo}
```

Comment:
This is unnecessary, mock is called with appropriate `--enablerepo` when necessary already.